src/components/Popover/Popover.tsx

```diff
@@ -83,28 +85,66 @@ export interface PopoverProps {
 // Letting this be implicit works in this project but fails in projects that use
 // generated *.d.ts files.
 
-export const Popover: React.FunctionComponent<PopoverProps> & {
+/*
+type PopoverComponentType = React.FunctionComponent<PopoverProps> & {
   Pane: typeof Pane;
   Section: typeof Section;
-} = function Popover({
-  activatorWrapper = 'div',
-  children,
-  onClose,
-  activator,
-  preventFocusOnClose,
-  active,
-  fixed,
-  ariaHaspopup,
-  preferInputActivator = true,
-  colorScheme,
-  zIndexOverride,
-  ...rest
-}: PopoverProps) {
+};
+*/
+
+export interface PopoverHandles {
+  forceReLayout(): void;
+}
+
+interface PopoverSubcomponents {
+  // TODO: How can we make these required props?
```

Comment:
This appears to be a difficult type to wrangle... and there major issues with me setting these to `optional (?)`, or is there another way I can type this to keep these subcomponents as required?

Reply:
Hey Curtis 👋🏽 I played around with this and ended up 🔥 the PopoverSubcomponents interface and trying [the second answer to this StackOverflow question](https://stackoverflow.com/questions/63453536/jsx-dot-notation-with-forwardref-typescript/68678190#68678190) and it works locally. 

```jsx
const PopoverNamespace = Object.assign(Popover, {Pane, Section});

export {PopoverNamespace as Popover};
```

I'm having trouble running `web` right now so I haven't been able to test on a consuming app (the comment on line 83 I'm guessing means this was a problem in the past 🤔 )